forte/data/ontology/core.py

```diff
@@ -635,5 +636,210 @@ def index_key(self) -> int:
         return self.tid
 
 
+class Grid:
+    """
+    Regular grid with a grid configuration dependent on the image size.
```

Comment:
rewrite this docstring to paragraphs, it is too long

Reply:
Split the docstring into paragraphs.